core/src/main/scala/neotypes/utils.scala

```diff
@@ -1,23 +1,32 @@
 package neotypes.utils
 
 import java.util.concurrent.CompletionException
+import java.util.function.{Consumer => JConsumer, Function => JFunction}
 
 object FunctionUtils {
```

Comment:
Are these utils expected to be used by users or are internal only?

Reply:
No, it's internal for `2.11` support only.